### ISSUE TITLE
Show drag distance doesn't support custom (dnd5e/other 5e based systems) diagonal rules

### ISSUE THREAD

**Issue by TheSirLancelot** [wsaunders1014/ShowDragDistance#88]
The tool does not utilize the settings for the system when determining diagonal measurements. Is there a setting that this is related to, or is this a bug?

**Comment by Geodragon9** [comment]
Yes. I've noticed for Starfinder that every other square should be 10' not 5', so it's not calculating diagonal properly.

**Comment by TheSirLancelot** [comment]
I found where it's handling the diagonal stuff. I'll fork and make the change myself. What's the starfinder game ID?

**Comment by TheSirLancelot** [comment]
> Yes. I've noticed for Starfinder that every other square should be 10' not 5', so it's not calculating diagonal properly.

I found where it's handling the diagonal stuff. I'll fork and make the change myself. What's the starfinder game ID?

Edit: unless the author is willing to merge my PR?
